config/rendered-items-assets-template.html

```diff
@@ -0,0 +1,144 @@
+        font-family: GH Guardian Headline;
+        font-style: italic;
+        font-weight: 400;
+        src: url("/assets/fonts/GHGuardianHeadline-RegularItalic.ttf");
+      }
+      @font-face {
+        font-family: GH Guardian Headline;
+        font-weight: 500;
+        src: url("/assets/fonts/GHGuardianHeadline-Medium.ttf");
+      }
+      @font-face {
+        font-family: GH Guardian Headline;
+        font-style: italic;
+        font-weight: 500;
+        src: url("/assets/fonts/GHGuardianHeadline-MediumItalic.ttf");
+      }
+      @font-face {
+        font-family: GH Guardian Headline;
+        font-weight: 600;
+        src: url("/assets/fonts/GHGuardianHeadline-Semibold.ttf");
+      }
+      @font-face {
+        font-family: GH Guardian Headline;
+        font-style: italic;
+        font-weight: 600;
+        src: url("/assets/fonts/GHGuardianHeadline-SemiboldItalic.ttf");
+      }
+      @font-face {
+        font-family: GH Guardian Headline;
+        font-weight: 700;
+        src: url("/assets/fonts/GHGuardianHeadline-Bold.ttf");
+      }
+      @font-face {
+        font-family: GH Guardian Headline;
+        font-style: italic;
+        font-weight: 700;
+        src: url("/assets/fonts/GHGuardianHeadline-BoldItalic.ttf");
+      }
+      @font-face {
+        font-family: Guardian Icons;
+        src: url("/assets/fonts/icons.otf");
+      }
+    </style>
+  </head>
+  <body>
+    <div style="font-family: Guardian Text Egyptian Web;">.</div>
+    <div style="font-family: Guardian Text Egyptian Web; font-weight: bold;">.</div>
+    <div style="font-family: Guardian Text Egyptian Web; font-style: italic;">.</div>
+    <div style="font-family: Guardian Text Egyptian Web; font-weight: italic; font-weight: 700;">.</div>
+    <div style="font-family: Guardian Text Sans Web;">.</div>
+    <div style="font-family: Guardian Text Sans Web; font-weight: bold;">.</div>
+    <div style="font-family: Guardian Text Sans Web; font-style: italic;">.</div>
+    <div style="font-family: Guardian Text Sans Web; font-style: italic; font-weight: bold;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 300;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 300; font-weight: bold;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 400;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 400; font-weight: bold;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 500;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 500; font-weight: bold;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 600;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 600; font-weight: bold;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 700;">.</div>
+    <div style="font-family: GH Guardian Headline; font-weight: 700; font-weight: bold;">.</div>
```

Comment:
A few duplicate `font-weight`s in here?

Reply:
the art of copy pasting .. will fix that later 👍